article/test/DCRFake.scala

```diff
@@ -31,4 +31,10 @@ class DCRFake(implicit context: ApplicationContext) extends renderers.DotcomRend
   ): Future[String] = {
     Future.successful("FakeRemoteRender has found you out if you rely on this markup!")
   }
+
+  override def getKeyEvents(ws: WSClient, page: LiveBlogPage, keyEvents: Seq[Block], filterKeyEvents: Boolean)(implicit
```

Comment:
I'm guessing we need this, and the new database files, because an existing test is making a fake request to DCR for getting key events?